singolo/index.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>SINGOLO</title>
+  </head>
+  <body>
+    <header class="header" id="header">
+      <div class="header-content">
+        <div class="header__logo">
+          <div class="burger-menu">
+            <div class="line"></div>
```

Comment:
is the _line_ child element for the burger menu? What about  _burger-menu__line_ or _element_

Reply:
fixed